files/default/neutron-ha-tool.py

```diff
@@ -404,9 +379,31 @@ def replicate_dhcp(qclient, noop=False):
     LOG.info("Added %s networks to DHCP agents", added)
 
 
+def migrate_l3_routers_from_agent(qclient, agent_id, target_ids, noop):
+    LOG.info("Querying agent_id=%s for routers to migrate away", agent_id)
+    router_id_list = list_routers_on_l3_agent(qclient, agent_id)
+
+    if not noop:
```

Comment:
There's one thing you're losing here: I think it was intended to actually have the logging when `noop` is set because it was used to know what the tool would do.